saga/memberrequestsaga.go

```diff
@@ -275,7 +275,7 @@ func (s *MemberRequestSaga) HandleEvent(event *eventstore.StoredEvent) ([]ep.Eve
 
 			}
 
-			if err := s.completeMemberChange(correlationID, causationID, memberChangeID, fmt.Sprintf("error updating member: %v, err")); err != nil {
+			if err := s.completeMemberChange(correlationID, causationID, memberChangeID, fmt.Sprintf("error updating member: %v, err", memberChangeID)); err != nil {
```

Comment:
The missing argument is "err" due to wrong closing quote position. Should be something like:

```
fmt.Sprintf("error updating member: %v", err)

If instead you also wanted to log the `memberChangeID` then please do this in another PR.